packages/server/database/migrations/20220121600507-addMoreRetroTemplates.ts

```diff
@@ -0,0 +1,667 @@
+import {R} from 'rethinkdb-ts'
+import {PALETTE} from '../../../client/styles/paletteV3'
+
+type PromptInput = {
+  question: string
+  description: string
+  templateId: string
+  sortOrder: number
+  promptColor?: string
+  promptId?: string
+}
+
+const promptColors = [
```

Comment:
Initially, I used `palettePickerOptions` which is used in the `addReflectTemplatePrompt` but noticed that some of the prompt colours were similar, so I created an array to make sure that they differ sufficiently.